test/jasmine/tests/polar_test.js

```diff
@@ -1136,7 +1143,9 @@ describe('Test polar interactions:', function() {
                 fig.layout.margin = {l: 50, t: 50, b: 50, r: 50};
 
                 if(s.patch) s.patch(fig);
-                nTraces = fig.data.length;
+                nTraces = fig.data.reduce(function(acc, trace) {
+                    return (trace.type === 'scatterpolargl') ? ++acc : acc;
```

Comment:
hmm, OK. I always find prefix `++` confusing, and perhaps because it encourages stuff like that `reduce` also often comes out confusing to me. I realize it's better perf this way, but that's irrelevant to a test.
`fig.data.filter(function(trace) { return trace.type === 'scatterpolargl'; }).length`?

Reply:
you're right, filter is better: b4700cabda7752b3d21f1277a37dd912664a2805